common/components/common/projects/AboutProjectDisplay.jsx

```diff
@@ -377,6 +381,14 @@ class AboutProjectDisplay extends React.PureComponent<Props, State> {
         />;
       });
     }
-}
+
+    _renderGroupIcon(group): ?Array<React$Node> {
+      if(!_.isEmpty(group.group_thumbnail)) {
+        return <div className="AboutProjects-group-image"><a href={url.section(Section.AboutGroup, {id: group.group_id})}><img src={group.group_thumbnail.publicUrl} alt={group.group_name + " Logo"} /></a></div>
+        } else {
+        return <div className="AboutProjects-group-image"><a href={url.section(Section.AboutGroup, {id: group.group_id})}><i className={Glyph(GlyphStyles.Users, GlyphSizes.X3)}></i></a></div>
+        }
```

Comment:
```suggestion
        return (
        <div className="AboutProjects-group-image">
          <a href={url.section(Section.AboutGroup, {id: group.group_id})}>
            !_.isEmpty(group.group_thumbnail) 
            ? <img src={group.group_thumbnail.publicUrl} alt={group.group_name + " Logo"} />
            : <i className={Glyph(GlyphStyles.Users, GlyphSizes.X3)}></i>
            </a>
          </div>
        )
        }
```

Can reduce some code duplication here

Reply:
I did that first, but the text positioning needs them to be separate elements; wrapping it all in an <a href> at the block level breaks that. 

It's fixable, of course - just look at how team members work - but it'll be a little more involved. Let me know.